src/client/components/MovieCard/index.jsx

```diff
@@ -1,3 +1,9 @@
+import { connect } from 'react-redux';
 import MovieCard from './MovieCard';
+import { fetchSingleMovieAction } from '../../actions/actions';
 
-export default MovieCard;
+const mapDispatchProps = dispatch => ({
```

Comment:
You can also (not required) use short record:
```js
const mapDispatchToProps = { fetchSingleMovie: fetchSingleMovieAction };
```

Reply:
I don't understand how can we omit "dispatch" here??
done as you proposed. ty